install

```diff
@@ -41,6 +41,7 @@ declare -A DEFAULT_OPTS=(
   [SKIP_METRICS]=false      # (default false) Skip installation of Swarmpit and Swarmprom
   [SKIP_SWARMPIT]=false     # (default false) Skip installation of Swarmpit
   [SKIP_SWARMPROM]=false    # (default false) Skip installation of Swarmprom
+  [MINIMAL_INSTALL]=false   # (default false) Use the minimal installation method
```

Comment:
Good idea.
What about `INSTALLATION_MODE=minimal` (options: `bare|minimal|full|...`)?
I'm still thinking about this, I'd like it to be clear what happens when you install Swarmlet.
So far I can think of these scenario's:
- `full` installation, so as is right now.
- `minimal` installation, assuming Docker and Docker Compose are installed already. (Also skip initialization of swarm mode?)
- `<another mode>` installation, making sure Traefik doesn't take over routing, ...

Reply:
> I'd like it to be clear what happens when you install Swarmlet.

Yep, this is good.

Soooooo, I guess it can get complicated. What is "full" when you have a single node cluster or a multi-node cluster? It is different. For example, I don't need Consul for single node, so is that still full?

I guess the complications come from packing in "defaults" that suppose a use case.  I don't think there is a "general case" here when each will be doing what works for them: single node, multi-node, no consul, using consul but with 3 replicas, etc. It is endless :)

I think a good simple default would be to assume the cluster is already up and configured how the end-user wants and then just handle git push passthrough and stack deploys. No traefik, consul or additionl projects. From my perspective that is the core offering of swarmlet (like Dokku) and that being rock solid and reliable would be an excellent thing in itself.